src/containers/TexterTodoList.jsx

```diff
@@ -226,38 +271,42 @@ const queries = {
           ownProps.location.query["org"] == "all" ||
           !ownProps.params.organizationId
             ? null
-            : ownProps.params.organizationId,
-        needsMessageFilter: {
-          messageStatus: "needsMessage",
-          isOptedOut: false,
-          validTimezone: true
-        },
-        needsResponseFilter: {
-          messageStatus: "needsResponse",
-          isOptedOut: false,
-          validTimezone: true
-        },
-        badTimezoneFilter: {
-          isOptedOut: false,
-          validTimezone: false
-        },
-        completedConvosFilter: {
-          isOptedOut: false,
-          validTimezone: true,
-          messageStatus: "messaged"
-        },
-        pastMessagesFilter: {
-          messageStatus: "convo",
-          isOptedOut: false,
-          validTimezone: true
-        },
-        skippedMessagesFilter: {
-          messageStatus: "closed",
-          isOptedOut: false,
-          validTimezone: true
-        }
+            : ownProps.params.organizationId
       }
     })
+  },
+  notifications: {
+    query: gql`
+      query getNotifications(
+        $userId: Int
+        $organizationId: String!
+      ) {
+        user(organizationId: $organizationId, userId: $userId) {
+          id
+          notifications {
+            ${assignmentQueryData}
+          }
+        }
+      }
+    `,
+    options: ownProps => {
```

Comment:
this gets re-run when `data` property is updated, so we can add conditions for when they come in and then not poll at all in scenarios where e.g. there aren't any todos